README.md

```diff
@@ -69,6 +69,10 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
 
-### `npm run install --save @fortawesome/fontawesome-free
+### `npm run install --save @fortawesome/fontawesome-free`
```

Comment:
How come this is still required? Does not `npm install` do the job when it's in package.json?

Reply:
Okay, I may have confused things here. How frequently should we be doing `npm install`? Admittedly, I forgot about it, so when I pulled from git, I expected it to just work as I knew the line was in package.json. Do we run `npm install` whenever we see package.json has been updated?

Reply:
That's exactly right yeah, once you npm install a package it goes into package.json and it gets installed with everything else with 'npm install' so yeah, if you spot packages.json (or package-lock.json) has been updated, then you updated by doing an 'npm install'.

Reply:
Sweet. So... I don't want to merge this, can I delete the pull request?